src/KogtervanFaculty.java

```diff
@@ -0,0 +1,39 @@
+public class KogtervanFaculty {
+
+  public String getFacultyName() {
+    return facultyName;
+  }
+
+  public void setFacultyName(String facultyName) {
+    this.facultyName = facultyName;
+  }
+
+  private String facultyName;
+
+  public KogtervanFaculty(String facultyName) {
+    this.facultyName = facultyName;
+  }
+
+
+
+
+  public void CompareStudents(KogtervanStudent firstStudent, KogtervanStudent secondStudent){
```

Comment:
Имена методов с маленькой буквы
```suggestion
  public void сompareStudents(KogtervanStudent firstStudent, KogtervanStudent secondStudent){
```